src/bin/bitmaskd.rs

```diff
@@ -53,10 +56,17 @@ async fn transfer(Json(transfer): Json<TransferRequest>) -> Result<impl IntoResp
     )
     .await?;
 
+    let contract = Contract::from_str(&transfer.asset_contract)?;
     let transfer_res = TransferResponse {
         consignment,
         psbt,
         disclosure,
+        declare_request: DeclareRequest {
```

Comment:
I added declare_request here to avoid compiling error, but I'm not sure if is a correct information.

Reply:
I'm doing this in another repo:

`let (consignment, psbt, disclosure, change, previous_utxo, new_utxo) = transfer_assets(
            &request.rgb_assets_descriptor_xpub,
            &request.blinded_utxo,
            request.amount,
            &request.asset_contract,
            request.asset_utxos,
        )
        .await?;

        let change: Vec<SealCoins> = serde_json::from_str(&change).unwrap();
        let change = change[0].clone();
        let previous_utxo: Vec<SealCoins> = serde_json::from_str(&previous_utxo).unwrap();
        let previous_utxo = previous_utxo[0].clone();

        let new_balance = previous_utxo.amount - change.amount;
        let new_outpoint = if new_balance == 0 {
            None
        } else {
            Some(NewOutpoint {
                outpoint: format!("{}:{}", &change.txid, &change.vout),
                disclosure: disclosure.clone(),
                balance: change.amount,
            })
        };
        let blinded_outpoint = Some(BlindedOutpoint {
            outpoint: new_utxo,
            consignment: consignment.clone(),
            balance: new_balance,
        });
        let declare_request = DeclareRequest {
            previous_utxo: format!("{}:{}", &previous_utxo.txid, &previous_utxo.vout),
            asset_id: Contract::from_str(&request.asset_contract)?
                .contract_id()
                .to_string(),
            new_outpoint,
            blinded_outpoint,
        };

        let result = TransferResponse {
            consignment,
            psbt,
            disclosure,
            declare_request,
        };`

Something like this must work